src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -9,133 +9,127 @@
 using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery;
 using Microsoft.CodeAnalysis.CSharp.LanguageServices;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Formatting;
 using Microsoft.CodeAnalysis.LanguageServices;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal sealed class CSharpSyntaxFactsService : CSharpSyntaxFacts, ISyntaxFactsService
+    internal sealed partial class CSharpSyntaxFactsServiceFactory
```

Comment:
This was just made a nested type and couple of redundant implementations were deleted (no new code added). However, by default GitHub shows many changes. I recommend reviewing this file with hidden whitespace changes: https://github.blog/2018-05-01-ignore-white-space-in-code-review/